test/unit/range/views/view_kmer_hash_test.cpp

```diff
@@ -206,3 +153,16 @@ TEST_F(kmer_hash_test, issue1643)
     auto k_mer_size_25_view = text_23_elements | seqan3::views::kmer_hash(seqan3::ungapped{25u});
     EXPECT_TRUE(k_mer_size_25_view.empty());
 }
+
+// https://github.com/seqan/seqan3/issues/1719
+TEST_F(kmer_hash_test, issue1719)
+{
+    uint64_t const expected = 0;
+    std::vector<seqan3::dna5> sequence{""_dna5};
+    auto v = sequence | seqan3::views::kmer_hash(seqan3::ungapped{25});
+    EXPECT_EQ(expected, v.size());
+
+    std::vector<seqan3::dna5> sequence2{"ACGATCGATCGTAGCTACTGAGC"_dna5};
```

Comment:
why such a long sequence?

```suggestion
    std::vector<seqan3::dna5> sequence2{"ACGATCGATCGTAGCTACTGAGC"_dna5};
    EXPECT_EQ(sequence2.size, 23); //???
```